src/components/report-technical-issue/report-technical-issue.yaml

```diff
@@ -2,11 +2,23 @@ params:
   - name: serviceCode
     type: string
     required: true
-    description: The URL-friendly service name which is passed to Contact Frontend
+    description: The URL-friendly service name which is passed to Contact Frontend.
   - name: language
     type: string
     required: false
     description: Either "en" for english or "cy" for welsh.
+  - name: classes
```

Comment:
Was this missing before? Should it be classList?

Reply:
You're right, it should be according to https://github.com/hmrc/hmrc-frontend/blob/master/src/components/report-technical-issue/template.njk#L4 but this is the only component which uses `classList` - the others all use `classes`.  It was me who wrote that previous version and I have no idea why I didn't follow the standard.

I'd suggest we remove `classList` and use `classes` instead to be consistent with the rest of the project and `govuk-frontend`.

Reply:
I'm OK with that.

Reply:
I've updated the template to follow the convention now.